pkg/service/role.go

```diff
@@ -264,6 +271,12 @@ func (s *roleService) Update(ctx context.Context, role *rolev3.Role) (*rolev3.Ro
 		}
 	}
 
+	//validate basic mandatory permissions that should be part of all custom roles
+	if !(utils.Contains(role.Spec.Rolepermissions, partnerR) &&
+		utils.Contains(role.Spec.Rolepermissions, organizationR)) {
+		return nil, fmt.Errorf("invalid role permissions, '%v', '%v', '%v' should be present ", partnerR, organizationR, projectR)
```

Comment:
We should probably drop the project check from here as we don't necessarily need it as of now.